files/settings-default.cfg

```diff
@@ -1,244 +1,709 @@
-# WARNING: Editing this file might have no effect, as these
-# settings are overwritten by your user settings file.
+# WARNING: Editing this file might have no effect, as these settings
+# are overwritten by your user settings file.  Your user settings file
+# varies with your operating system:
+#
+# Linux: $HOME/.config/openmw
+# Mac: $HOME/Library/Preferences/openmw
+# Windows: C:\Users\Username\Documents\my games\openmw
+#   This path may vary depending on your installation hard drive, your
+#   Windows username, and your default language.
+#
+# Additionally, the user settings file is often written to disk when
+# exiting OpenMW, so comments and changes to that file may also be
+# discarded after running OpenMW.  While most changes to the file will
+# reflect setting changes made in game, some settings can have a wider
+# range of values in the settings file than the GUI settings widgets
+# allow.  You may want to exercise some caution and backup this file
+# when editing it by hand.
```

Comment:
I'd argue that crashing due to config values shouldn't be possible.   Exactly how to determine what's a "valid" value is still an open question.  At one point I considered the limits in the Options GUI to be authoritative, but there are several settings where that's seriously counter-productive (e.g. viewing distance).  In the end, I think values that crash or produce ridiculous results should be disallowed.  I'm still investigating what those values are for some settings.   See my recent comment on "bugs".